core/idl_types.h

```diff
@@ -19,7 +19,11 @@
 
 #include "core/idl_base.h"
 
+struct IDLBoolean final : public IDLBaseHelper<bool> {};
 struct IDLDouble final : public IDLBaseHelper<double> {};
+struct IDLFloat final : public IDLBaseHelper<float> {};
+struct IDLLong final : public IDLBaseHelper<long> {};
+struct IDLShort final : public IDLBaseHelper<short> {};
```

Comment:
ditto

Reply:
I checked platform when return value.
Could you review it?